docs/installation.rst

```diff
@@ -134,6 +134,11 @@ IRC
 
     - IRC server you wish to connect to (default: ``chat.freenode.net``)
 
+- **IRC_MAX_MESSAGE_LENGTH**:
```

Comment:
If you are going to add this to the available configs, you'll need to add `IRC_MAX_MESSAGE_LENGTH` to the available options in the `env.example` file.